### PR TITLE
Fsc Maseeval scripts

### PR DESCRIPTION
**Pull request by siddhu001**

### CODE REVIEW

**Comment by sw005320** [comment]
Can you merge the latest master?
test_debian9 should be passed then.

**Comment by siddhu001** [comment]
> Can you merge the latest master? test_debian9 should be passed then.

Done!

File: egs2/README.md [hunk 1]
```diff
@@ -23,6 +23,8 @@ See: https://espnet.github.io/espnet/espnet2_tutorial.html#recipes-using-espnet2
 | dirha_wsj               | Distant-speech Interaction for Robust Home Applications                                 | Multichannel ASR        | EN                    | https://dirha.fbk.eu/, https://github.com/SHINE-FBK/DIRHA_English_wsj                                        |              |
 | dns_ins20               | Deep Noise Suppression Challenge – INTERSPEECH 2020                                     | SE                      | 7 languages + singing | https://www.microsoft.com/en-us/research/academic-program/deep-noise-suppression-challenge-interspeech-2020/ |              |
 | fsc                     | Fluent Speech Commands Dataset                                                          | SLU                     | EN                    | https://fluent.ai/fluent-speech-commands-a-dataset-for-spoken-language-understanding-research/               |              |
+| fsc_unseen              | Fluent Speech Commands Dataset MASE Eval splits                                         | SLU                     | EN                    | https://github.com/maseEval/mase                                                                             |              |
```
Comment (sw005320) [inline egs2/README.md:26]: What is MASE?

File: egs2/fsc_unseen/asr1/README.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+## Environments
+- date: `Mon Oct 11 13:11:36 2021 -0400`
+- python version: `3.9.5 (default, Jun  4 2021, 12:28:51) [GCC 7.5.0]`
+- espnet version: `espnet 0.10.3a2`
+- pytorch version: `pytorch 1.8.1+cu102`
+- Git hash: `4e7d2ba3510463ae744d1a6d98f18388ad929a9d`
+  - Commit date: `Mon Nov 8 16:28:44 2021 -0500`
+- Pretrained Model
+  - Zenodo: https://zenodo.org/record/5655832
```
Comment (sw005320) [inline egs2/fsc_unseen/asr1/README.md:12]: How about adding a model to HF hub as well and add an HF hub link?

File: egs2/fsc_challenge/asr1/README.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+## Environments
+- date: `Sun Oct  3 22:25:25 EDT 2021`
+- python version: `3.8.11 (default, Aug  3 2021, 15:09:35)  [GCC 7.5.0]`
+- espnet version: `espnet 0.10.3a3`
+- pytorch version: `pytorch 1.9.0+cu102`
+- Git hash: `a1a55e1eef2a74d2b8580d8071ce5229e7fa654c`
+  - Commit date: `Mon Nov 8 23:56:06 2021 -0500`
+- Pretrained Model
+  - Zenodo: https://zenodo.org/deposit/5656007
```
Comment (sw005320) [inline egs2/fsc_challenge/asr1/README.md:12]: How about adding a model to HF hub as well and add an HF hub link?

**Review by sw005320 — COMMENTED** [review]
It’s better to explain about the unseen and challenge setups (in fsc_xxx/asr1/README.md ?) by referring your paper.

File: egs2/fsc_challenge/asr1/conf/tuning/train_asr_transformer_adam.yaml [hunk 4]
```diff
@@ -0,0 +1,31 @@
+# network architecture
```
Comment (ftshijt) [inline egs2/fsc_challenge/asr1/conf/tuning/train_asr_transformer_adam.yaml:1]: Can you combine some very similar configs? like this one and the `train_asr_transformer.yaml`, `train_asr_transformer_specaug.yaml`

File: egs2/README.md (hunk 1 above)
Comment (siddhu001) [reply egs2/README.md:26]: MASE (Multi-Aspect Subtask Evaluation) is the name that we gave to our methodology for generating splits as shown here - https://maseeval.github.io/

Comment (siddhu001) [reply egs2/README.md:26]: I will explain a bit more about this in the respective Readme for each split

File: egs2/fsc_unseen/asr1/README.md (hunk 2 above)
Comment (siddhu001) [reply egs2/fsc_unseen/asr1/README.md:12]: Will do!

File: egs2/fsc_challenge/asr1/README.md (hunk 3 above)
Comment (siddhu001) [reply egs2/fsc_challenge/asr1/README.md:12]: Will do!

File: egs2/fsc_challenge/asr1/conf/tuning/train_asr_transformer_adam.yaml (hunk 4 above)
Comment (siddhu001) [reply egs2/fsc_challenge/asr1/conf/tuning/train_asr_transformer_adam.yaml:1]: Sounds good! I will try to combine some of them and remove some other configs not used.

**Comment by siddhu001** [comment]
@ftshijt I have take care of all the feedback on the pull request. Weirdly, the CI checks are failing because it cannot access the models on hugging face - 
requests.exceptions.HTTPError: 500 Server Error: Internal Server Error for url: https://huggingface.co/api/models/espnet/kamo-naoyuki-mini_an4_asr_train_raw_bpe_valid.acc.best

**Comment by sw005320** [comment]
Thanks a lot!
The CI check was fully passed!

**Pull request merged**